client/src/templates/Challenges/redux/code-storage-epic.js

```diff
@@ -150,7 +150,13 @@ function loadCodeEpic(action$, state$) {
             // This was pushed to production with https://github.com/freeCodeCamp/freeCodeCamp/pull/43023
             foundChallengeFile = codeFound[challengeFile.fileKey];
```

Comment:
Seeing as we are just purging unwanted formats, why not remove this "old code-storage format" stuffs too?